CodingConventions.md

```diff
@@ -0,0 +1,145 @@
+# Overview
+
+We would like to be more focused on following the coding conventions popular in .NET community.
+Currently the coding style in the driver code is not very consistent, but the goal is to improve it
+by following coding convention rules.
+
+# Coding conventions
+
+## Writing code
+
+### Static variables
+
+#### Internal or private static member
+
+Use `s_` prefix + CamelCase, eg. `s_someVariable`.
```

Comment:
Could you change `CamelCase` to `camelCase`.

Reply:
What I mean is that the `camelCase` starts with lowercase letter whereas `PascalCase` starts with uppercase letter. Thanks to changing the `camelCase` word in the text it would also follow it's own convention :)

Reply:
ok, I changed it